packages/video_player/video_player/README.md

```diff
@@ -125,6 +126,34 @@ documentation in order to give a more elaborate overview of the API.
 
 This is not complete as of now. You can contribute to this section by [opening a pull request](https://github.com/flutter/plugins/pulls).
 
+### Data Sources
+
+You can provide a video from different data sources using different constructors.
+
+- `VideoPlayerController.asset` loads a video from your Flutter assets,
+- `VideoPlayerController.file` loads a video from a local file,
+- `VideoPlayerController.network` loads a video from the network, and
+- (**Android Only**) `VideoPlayerController.contentUri` loads a video from a provided content-uri.
+
+You cannot pass a YouTube video URI into the `VideoPlayerController.network` constructor.
+Functionality to decode YouTube URIs is not provided by this plugin.
+
+To learn more about the limitations of `VideoPlayerController` constructors, see the [`Constructors` documentation](https://pub.dev/documentation/video_player/latest/video_player/VideoPlayerController-class.html#constructors).
+
+For specific information on supported video formats on each platform, please refer back to the [Supported Formats](##supported-formats) section.
+
+### Looping
+
+You can set the looping behavior on your `_controller` (instance of `VideoPlayerController`) by calling
+`_controller.setLooping`. `setLooping` takes a `bool` value indicating whether the video will loop after being played once. Looping behavior defaults to false.
+
+You can also set the `_controller` to start the video at a specific timestamp by calling `_controller.seekTo`.
+`seekTo` takes a `Duration` object (`moment`) indicating the timestamp from where the video will resume the next time it is played.
+
+Setting looping to true and providing a to the `seekTo` method will start the video again from the provided `moment`.
```

Comment:
There's a word or phrase missing here, so I'm not sure what exactly this is trying to say, but as above I'm not sure what the relationship between seeking and looping is here.